scss/styles.scss

```diff
@@ -5,3 +5,12 @@
   width: 100%;
   height: 100%;
 }
+
+.card {
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
+  background: $white;
+  display: flex;
+  flex-flow: row wrap;
```

Comment:
Why card have flex wrap? it is necessary?

Reply:
i think is unnecessary

Reply:
ok